### ISSUE TITLE
Symlinked custom component does not work

### ISSUE THREAD

**Issue by Jpsy** [py-smart-gardena/hass-gardena-smart-system#14]
The docs contain a manual installation method that uses a symlink:  
```
cd <path>/<to>/<your>/<config>
git clone https://github.com/py-smart-gardena/hass-gardena-smart-system.git
mkdir custom_components (if not exist)
cd custom_components
ln -s ../hass-gardena-smart-system/custom_components/gardena_smart_system
```
This can be very helpful as the cloned path can contain a full git repo to work on while the symlink in custom_components points to the sub-path that contains the actual component.

So I tried this setup today and found that HA does completely ignore any symlinked paths in custom_components. 

Here is my current symlink setup (NOT working):

![image](https://user-images.githubusercontent.com/1806058/83440245-e4b4b180-a444-11ea-864e-de725b8af21e.png)

I also tried to install the component using HACS and then replaced the resulting sub-path in custom_components by a symlink. Same result: HA does not find the component after a restart.

If this should be true, we should update the docs and remove the symlink example.

**Comment by Jpsy** [comment]
Testing through this issue for hours I found the explanation just minutes after posting the issue: 

I created the symlink not directly in an SSH shell on hass.io, but in a Mac terminal on the mounted hass.io file system. I thought the linking information in the resulting symlinks would be identical, but obviously it is not. Creating the symlink directly on the HA file system using an SSH shell worked immediately.

Sorry for raising unnecessary dust, guys! 
Issue can be closed.
